tests/test_testbench_retry.py

```diff
@@ -786,6 +786,78 @@ def test_grpc_return_error_after_bytes(self):
         self.assertEqual(blob.bucket, "projects/_/buckets/bucket-name")
         self.assertEqual(blob.size, 2 * UPLOAD_QUANTUM)
 
+    def test_grpc_bidiwrite_return_error_after_bytes(self):
+        # Setup two after-bytes errors to test injecting failures in
```

Comment:
Do we need a test for initial-response errors as well? Or are existing tests sufficient for this?

Reply:
There are existing tests for initial-response errors. It doesn't hurt to add an extra check here, updated! Will be merging shortly